ansible/roles/fanout/library/port_config_gen.py

```diff
@@ -150,9 +150,9 @@ def _read_from_port_config(filepath):
         port_config = {}
         with open(filepath) as fd:
             lines = fd.readlines()
-            data_index = 0
-            while lines[data_index].startswith("#"):
-                data_index = data_index + 1
+            data_index = len(lines)
+            while not lines[data_index-1].startswith("#"):
```

Comment:
what if there are comments at the end/middle of the file?

Reply:
Hi @roy-sror , originally, it was just designed to handle the blank line before header line. Anyway, your comments was reasonable. So I updated the commit to handle the blank line and comments in the middle or at the end.